geth/account/importer_test.go

```diff
@@ -0,0 +1,81 @@
+package account
```

Comment:
Once again a whit-box. Here maybe the wiki document is misleading. I would interpret it as `foo` should not test `bar`, but `foo_test` should. Here also the test code won't be linked into the binary.

Corrections are welcome if I'm wrong with my thoughts.

Reply:
Sorry, I'm afraid you're wrong.
From https://golang.org/pkg/testing/  : 
```To write a new test suite, create a file whose name ends _test.go that contains the TestXxx functions as described here. Put the file in the same package as the one being tested. The file will be excluded from regular package builds but will be included when the “go test” command is run.```

Reply:
Good to know, thanks.